.github/workflows/network-deploy.yml

```diff
@@ -84,11 +84,6 @@ on:
         required: false
         type: string
         default: "false"
-      ref:
```

Comment:
I would like to keep this please, otherwise the values file needs to be in master in order to test a deployment.

Reply:
unless the github.ref below does the same thing in that case all gucci

Reply:
yesss workflow dispatch actions have this by default, we don't need an extra input of our own
<img width="140" alt="image" src="https://github.com/user-attachments/assets/c0274bc1-a77a-4950-bc69-e9b198ec2601" />